.github/workflows/ci-docker.yml

```diff
@@ -0,0 +1,32 @@
+name: Docker CI
+
+on:
+  pull_request:
+    branches: ['main']
+    paths: ['Dockerfile','cmd/**','docs/**','internal/**','go.*','.github/workflows/ci-docker.yml']
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: blinklabs/snek
```

Comment:
This is fine for here, but publishing will need to be `blinklabs-io/snek` when we actually push.

Reply:
I fixed this anyway